Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,81 @@
+var humanChoice = "";
+var humanScore = 0;
+var computerScore = 0;
+
+
+$('.choice').click(function() {
+   humanChoice = this.id;
```

Comment:
Ahh using the `this` keyword to avoid using wrapper functions, *VERY* fancy, I'm super impressed 👏 👏